### ISSUE TITLE
Click events on working on charts

### ISSUE THREAD

**Issue by unaor** [primefaces/primeng#1487]
Im trying to capture the pie chart click event as shown by the documentation
`<p-chart type="line" [data]="data" (onDataSelect)="selectData($event)"></p-chart>`

However im getting the error
Error in ./UIChart class UIChart - inline template:2:64 caused by: Cannot read property 'intersect' of undefined
I noticed that the error is on chart.js line 24

```
UIChart.prototype.onCanvasClick = function (event) {
        if (this.chart) {
            var element = this.chart.getElementAtEvent(event);
            var dataset = this.chart.getDatasetAtEvent(event);
            if (element && element[0] && dataset) {
                this.onDataSelect.emit({ originalEvent: event, element: element[0], dataset: dataset });
            }
        }
    };
```
`var items = options.intersect ? getIntersectItems(chart, position) : getNearestItems(chart, position, false);`

im using these versions
"chart.js": "^2.4.0"
"primeng": "^1.0.0"

**Comment by unaor** [comment]
I just verified it and its working with chart.js 2.3

**Comment by unaor** [comment]
Someone was able to look at the issue?

**Comment by TonkWorks** [comment]
The issue is in the underlying chart.js 2.4.0 -  It was fixed here (https://github.com/chartjs/Chart.js/issues/3606) but has not yet been released.

For now, I would use chart.js 2.3.0 until the next version of chart.js comes out.

**Comment by jiangyaqi2** [comment]
It's bug?

**Comment by umens** [comment]
This issue can be closed since chart js has been updated to 2.5.0 and this release fix the bug.

**Comment by sudheerj** [comment]
The issue is related to external charjs library.This has been fixed in 2.5.0 version and working good too.So we can close this issue now.
